manifests/04_roles.yaml

```diff
@@ -36,24 +36,24 @@ rules:
       - watch
   - apiGroups:
       - rbac.authorization.k8s.io
-      resources:
-        - ClusterRole
-        - RoleBinding
-        - Role
-      verbs:
-        - list
-        - get
-        - create
-        - update
-        - watch
+    resources:
+      - ClusterRole
+      - RoleBinding
+      - Role
+    verbs:
+      - list
+      - get
+      - create
+      - update
+      - watch
```

Comment:
please fix in the commit that introduced these changes :)

Reply:
Done